pombola/south_africa/static/js/mp-profiles-live-filter.js

```diff
@@ -39,7 +39,19 @@ var showFilterResults = function showFilterResults(){
     return $results;
 };
 
+var analyticsEventSent = false;
+
 var filterProfiles = function filterProfiles(searchTerm, filterer){
+
+    if (!analyticsEventSent) {
```

Comment:
Maybe this is crazy, but you could achieve this "fire once" functionality without the global `analyticsEventSent` variable by hooking a `$.one()` listener onto the `js-mp-profiles-live-filter:updated` event that `filterProfiles` fires once it’s finished filtering the list. Off the top of my head, I think this would look like:

```javascript
$('body').one('js-mp-profiles-live-filter:updated', function(){ … });
```